packages/verification-vue-component/src/assets/img/error_document.svg

```diff
@@ -0,0 +1,23 @@
+</g>
+<path fill-rule="evenodd" clip-rule="evenodd" d="M92 138C92 126.954 100.954 118 112 118C123.04 118.013 131.987 126.96 132 138C132 149.046 123.046 158 112 158C100.954 158 92 149.046 92 138ZM121.604 146.414C121.757 145.831 121.583 145.212 121.15 144.793L114.65 138.295C114.572 138.217 114.528 138.111 114.528 138.001C114.528 137.89 114.572 137.785 114.65 137.707L121.15 131.207C121.6 130.793 121.788 130.165 121.638 129.572C121.488 128.978 121.025 128.515 120.432 128.364C119.839 128.214 119.211 128.4 118.797 128.85L112.297 135.35C112.218 135.428 112.112 135.472 112.002 135.472C111.891 135.472 111.785 135.428 111.707 135.35L105.207 128.85C104.556 128.198 103.501 128.197 102.849 128.847C102.197 129.498 102.196 130.553 102.847 131.205L109.347 137.705C109.425 137.783 109.469 137.889 109.469 137.999C109.469 138.11 109.425 138.215 109.347 138.293L102.847 144.793C102.196 145.444 102.196 146.499 102.847 147.15C103.502 147.79 104.548 147.79 105.203 147.15L111.703 140.65C111.867 140.489 112.13 140.489 112.293 140.65L118.793 147.15C119.212 147.583 119.831 147.757 120.414 147.604C120.997 147.452 121.452 146.997 121.604 146.414Z" fill="#EE4266"/>
+<defs>
+<filter id="filter0_dd" x="0" y="0" width="124" height="162" filterUnits="userSpaceOnUse" color-interpolation-filters="sRGB">
+<feFlood flood-opacity="0" result="BackgroundImageFix"/>
+<feColorMatrix in="SourceAlpha" type="matrix" values="0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 127 0"/>
+<feOffset dy="1"/>
+<feGaussianBlur stdDeviation="2"/>
+<feColorMatrix type="matrix" values="0 0 0 0 0.12549 0 0 0 0 0.2 0 0 0 0 0.27451 0 0 0 0.04 0"/>
+<feBlend mode="normal" in2="BackgroundImageFix" result="effect1_dropShadow"/>
+<feColorMatrix in="SourceAlpha" type="matrix" values="0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 127 0"/>
+<feOffset dy="2"/>
+<feGaussianBlur stdDeviation="6"/>
+<feColorMatrix type="matrix" values="0 0 0 0 0.12549 0 0 0 0 0.2 0 0 0 0 0.27451 0 0 0 0.09 0"/>
+<feBlend mode="normal" in2="effect1_dropShadow" result="effect2_dropShadow"/>
+<feBlend mode="normal" in="SourceGraphic" in2="effect2_dropShadow" result="shape"/>
+</filter>
+</defs>
+</svg>
```

Comment:
This svg can be compressed by 34.3% with https://vecta.io/nano, you need to readd `viewBox="0 0 142 168"` after though.